src/plugins/data_source/server/client/data_source_client.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Logger, OpenSearchClient, SavedObjectsClientContract } from 'src/core/server';
+
+/**
+ * Represents an OpenSearch cluster API client created by the platform.
+ * It allows to call API on behalf of user defined in "data source" saved object
```

Comment:
can you help me understand the purpose of 'user' in the 'data source' saved object? I didn't see it in the current implementation.

Reply:
I mean the user/credential associated with data source